Source/Charts/Charts/ChartViewBase.swift

```diff
@@ -475,41 +447,28 @@ open class ChartViewBase: NSUIView, ChartDataProvider, AnimatorDelegate
     /// Highlights the value selected by touch gesture.
     @objc open func highlightValue(_ highlight: Highlight?, callDelegate: Bool)
     {
-        var entry: ChartDataEntry?
-        var h = highlight
-        
-        if h == nil
-        {
-            _indicesToHighlight.removeAll(keepingCapacity: false)
-        }
-        else
-        {
-            // set the indices to highlight
-            entry = _data?.entryForHighlight(h!)
-            if entry == nil
-            {
-                h = nil
+        var high = highlight
+        guard
+            let h = high,
+            let entry = _data?.entryForHighlight(h)
+            else {
```

Comment:
sorry about missing these code style, { another line

Reply:
I thought we agreed this was the style for multiline guard statements?

Reply:
oh is it? sorry don't remember.